packages/@okta/vuepress-site/docs/reference/api-overview/index.md

```diff
@@ -79,7 +79,34 @@ Okta will derive client request context directly from the HTTP request headers a
 
 Okta supports the standard `User-Agent` HTTP header to identify the user's browser or application. Always send a `User-Agent` string to uniquely identify your client application and version such as `Oktaprise/1.1`.
 
-> If your application is acting as a gateway or proxy, you should forward the `User-Agent` of the originating client with your API requests.
+> **Note:** If your application is acting as a gateway or proxy, you should forward the `User-Agent` of the originating client with your API requests.
+
+### Format a User-Agent string for native mobile applications
```

Comment:
Just noticed that this says "for native mobile applications" 

It could just say "Format a User-Agent string" since the the paragraph below it is not specific to native mobile applications.